stellar/contract.js

```diff
@@ -1,40 +1,57 @@
 const { loadConfig, printInfo, saveConfig } = require('../evm/utils');
-const { stellarCmd, getNetworkPassphrase, addBaseOptions, getWallet, broadcast } = require('./utils');
+const { stellarCmd, getNetworkPassphrase, addBaseOptions, getWallet, broadcast, serializeValue, addressToScVal } = require('./utils');
 const { getChainConfig, addOptionsToCommands } = require('../common');
 const { prompt } = require('../common/utils');
 require('./cli-utils');
 
 const MAX_INSTANCE_TTL_EXTENSION = 535679;
 
-async function submitOperation(chain, _, contract, operation, _args, options) {
-    const pauseOperation = operation;
+async function submitOperation(chain, _, contract, operation, options) {
     const wallet = await getWallet(chain, options);
-    const callOperation = await contract.call(pauseOperation);
-    const returnValue = await broadcast(callOperation, wallet, chain, `${pauseOperation} performed`, options);
+    const callOperation = await contract.call(operation);
 
-    if (returnValue.value()) {
-        printInfo('Return value', returnValue.value());
-    }
+    return broadcast(callOperation, wallet, chain, `${operation}`, options);
+}
+
+async function commonOperation(chain, _, contract, operation, options) {
+    const returnValue = await submitOperation(chain, _, contract, operation, options);
+
+    printInfo('Return value', serializeValue(returnValue.value()));
+}
+
+async function owner(chain, _, contract, operation, options) {
+    const returnValue = await submitOperation(chain, _, contract, operation, options);
+    const ownerScAddress = returnValue.value();
+
+    printInfo('Owner', Address.fromScAddress(ownerScAddress).toString());
 }
 
-async function isPausedCmd(chain, _, contract, _args, options) {
-    const operation = _args;
-    await submitOperation(chain, _, contract, operation, [], options);
+async function transferOwnership(chain, _, contract, args, options) {
+    const newOwner = args;
+    const wallet = await getWallet(chain, options);
+    const operation = contract.call('transfer_ownership', addressToScVal(newOwner));
+
+    await broadcast(operation, wallet, chain, 'transfer_ownership', options);
 }
 
-async function pauseCmd(chain, _, contract, _args, options) {
-    const operation = _args;
-    await submitOperation(chain, _, contract, operation, [], options);
+async function operator(chain, _, contract, operation, options) {
+    const returnValue = await submitOperation(chain, _, contract, operation, options);
+    const operatorScAddress = returnValue.value();
+
+    printInfo('Operator', Address.fromScAddress(operatorScAddress).toString());
 }
 
-async function unpauseCmd(chain, _, contract, _args, options) {
-    const operation = _args;
-    await submitOperation(chain, _, contract, operation, [], options);
+async function transferOperatorship(chain, _, contract, args, options) {
+    const newOperator = args;
+    const wallet = await getWallet(chain, options);
```

Comment:
```suggestion
async function transferOperatorship(chain, _, contract, newOperator, wallet, options) {
```
`wallet` should be an arg

Reply:
updated